featurebyte/service/observation_table.py

```diff
@@ -48,43 +47,12 @@ class PointInTimeStats:
     most_recent: str
 
 
-async def get_point_in_time_stats(
-    db_session: BaseSession,
-    destination: TableDetails,
-) -> PointInTimeStats:
-    """
-    Get the most recent point in time from the destination table.
-
-    Parameters
-    ----------
-    db_session: BaseSession
-        Database session
-    destination: TableDetails
-        Table details of the materialized table
-
-    Returns
-    -------
-    str
-    """
-    res = await db_session.execute_query(
-        get_least_and_most_recent_point_in_time_sql(
-            destination=destination,
-            source_type=db_session.source_type,
-        )
-    )
-    least_recent_point_in_time = res.iloc[0, 0]  # type: ignore[union-attr]
-    most_recent_point_in_time = res.iloc[0, 1]  # type: ignore[union-attr]
-
-    def _convert_ts_to_str(timestamp_str: str) -> str:
-        current_timestamp = pd.Timestamp(timestamp_str)
-        if current_timestamp.tzinfo is not None:
-            current_timestamp = current_timestamp.tz_convert("UTC").tz_localize(None)
-        current_timestamp = current_timestamp.isoformat()
-        return cast(str, current_timestamp)
-
-    most_recent_point_in_time_str = _convert_ts_to_str(most_recent_point_in_time)
-    earliest_str = _convert_ts_to_str(least_recent_point_in_time)
-    return PointInTimeStats(most_recent=most_recent_point_in_time_str, least_recent=earliest_str)
+def _convert_ts_to_str(timestamp_str: str) -> str:
+    current_timestamp = pd.Timestamp(timestamp_str)
```

Comment:
Not a new issue, but this `current_timestamp` variable name is a bit misleading - I thought it refers to the current time. Might want to change it to something like just `timestamp` or `timestamp_obj`.

Reply:
will rename!